powersimdata/input/converter/pypsa_to_grid.py

```diff
@@ -517,4 +515,4 @@ def _translate_pnl(self, pnl, key):
     @property
     def __class__(self):
         """If anyone asks, I'm a Grid object!"""
-        return Grid
+        return _grid.Grid
```

Comment:
I have overwritten `__class__` in bcf3bedf0057813725cf5ac3edc2594a65ae153f in order to be able use the grid equality in the `test_pypsa_to_grid_module` (see [test_import_exported_network](https://github.com/Breakthrough-Energy/PowerSimData/blob/5b6b3a34e301254b0ccdc40172daabf28fd57940/powersimdata/input/converter/tests/test_pypsa_to_grid.py#L91)). Not sure we still need to do that now that the `Grid` class` can handle `europe_tub` and if there is a way to re-write the test.

Reply:
This is handled in the latest commit [0ae9737](https://github.com/Breakthrough-Energy/PowerSimData/pull/707/commits/0ae9737a66755dfdf9521c829788144a87d20800) (although I had to temporarily hack the other tests so it would work).

Reply:
Thanks!